### ISSUE TITLE
[question] object.restore doesn't touch updated_at column

### ISSUE THREAD

**Issue by abitdodgy** [rubysherpas/paranoia#264]
It appears that `restore` method doesn't change the `updated_at` timestamp. This is probably by design, so I would like to understand the reasoning.

Potentially, this is an issue when using fragment caching. For example, an object can be destroyed and then rendered and cached in a separate view. After it's restored the cache fragment will be from its destroyed state.

It's very easy to work around this, but I'm just curious.

**Comment by radar** [comment]
I agree that it should change the updated_at time stamp. Please submit a fix for this.

**Comment by abitdodgy** [comment]
Sounds good! I'll send one!

**Comment by abitdodgy** [comment]
I still plan on issuing a pull request for this issue. I just haven't had a chance to yet.

**Comment by abitdodgy** [comment]
It looks like @leonardoalifraco beat me to it.
